src/Configuration/Cache.php

```diff
@@ -101,6 +102,35 @@ class Cache extends ConfigurationAnnotation
      */
     private $staleIfError;
 
+    public function __construct(
+        $values = [],
+        $expires = null,
+        $maxage = null,
+        $smaxage = null,
+        $public = false,
+        $mustRevalidate = false,
+        $vary = [],
+        $lastModified = null,
+        $Etag = null,
```

Comment:
The capital `E` looks odd.

Reply:
I'm agree with you but i'm annoyed because on the documentation, we suggest to use "Etag" and not "etag" for the annotation Cache. 
So, I can transform Etag to etag, but I need to add a information on the documentation to inform that for attributes, we use etag and not Etag. What do you think about that ?